modules/ztp-enabling-assisted-installer-service-on-bare-metal.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-preparing-the-hub-cluster.adoc
+
+[id="enabling-assisted-installer-service-on-bare-metal_{context}"]
+= Enabling the assisted service and updating AgentServiceConfig on the hub cluster
+
+{rh-rhacm-first} uses the assisted service to deploy {product-title} clusters. The assisted service is deployed automatically when you enable the MultiClusterHub Operator with Central Infrastructure Management (CIM). When you have enabled CIM on the hub cluster, you then need to update the `AgentServiceConfig` custom resource (CR) with references to the ISO and RootFS images that are hosted on the mirror registry HTTP server.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in to the hub cluster as a user with `cluster-admin` privileges.
+
+* You have enabled the assisted service on the hub cluster. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.6/html/multicluster_engine/multicluster_engine_overview#enable-cim[Enabling CIM].
+
+.Procedure
+
+. Update the `AgentServiceConfig` CR by running the following command:
++
+[source,terminal]
+----
+$ oc edit AgentServiceConfig
+----
+
+. Add the following entry to the `items.spec.osImages` field in the CR:
++
+[source,yaml,subs="attributes+"]
+----
+- cpuArchitecture: x86_64
+    openshiftVersion: "{product-version}"
+    rootFSUrl: https://<mirror_registry>/<path>/rhcos-live-rootfs.x86_64.img
```

Comment:
I wouldn't call this mirror_registry. 
It's just a http server where the image can be downloaded from. Note this is not a container image where usually resides in a container image registry.
maybe http_host or simply host would be better.